gtsfm/evaluation/metrics_report.py

```diff
@@ -372,7 +372,7 @@ def get_html_header() -> str:
 def generate_metrics_report_html(
     metrics_groups: List[GtsfmMetricsGroup],
     html_path: str,
-    other_pipelines_metrics_groups: Dict[str, List[GtsfmMetricsGroup]],
+    other_pipelines_metrics_groups: Optional[Dict[str, List[GtsfmMetricsGroup]]],
```

Comment:
Can we just pass an empty dict?

Reply:
Using a mutable data structure (empty dict) as an arg leads to some nasty bugs, so i'd avoid that if possible

Reply:
im just fixing the type hint here, we are already passing a `None` somewhere.